src/components/dashboard/dashboardv1.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState, useEffect } from "react";
```

Comment:
hmmm I think these dashboardv1 files should up uppercase...I changed them to be that. 😭